Objects/listobject.c

```diff
@@ -2145,12 +2145,21 @@ list.sort
     key as keyfunc: object = None
     reverse: bool(accept={int}) = False
 
-Stable sort *IN PLACE*.
+Sort the list in ascending order.
+
+The sort is in-place (i.e. the list itself is modified) and stable (i.e. the
+order of two equal elements is maintained).
+
+A key function can be supplied to customize the sorting behavior. The key
+function is applied to each list element, and the elements are sorted
+according to the values returned by the key function.
```

Comment:
More information in fewer words:
If a key function is given, apply it once to each list item and sort them, ascending or descending, according to their function values.

Reply:
Terry's version is better.